### PR TITLE
Add ArtBlocks (https://artblocks.io) to phishing config

### PR DESCRIPTION
**Pull request by jakerockland**

### CODE REVIEW

**Comment by devenblake** [comment]
Looks fine to me as long as the workflows go through okay.

**Review by gravelcycles — DISMISSED** [review]

**Review by gravelcycles — APPROVED** [review]

File: src/config.json [hunk 1]
```diff
@@ -909,6 +910,9 @@
     "metapass.fyi"
   ],
   "blacklist": [
+    "ártblocks.net",
+    "ártblocks.io",
+    "ártblocks.com",
```
Comment (gravelcycles) [inline src/config.json]: I think `ártblocks.net` should be `xn--rtblocks-7ya.net` in order to handle the tilde.

Comment (gravelcycles) [reply src/config.json]: @jakerockland

**Review by gravelcycles — CHANGES_REQUESTED** [review]

File: src/config.json (hunk 1 above)
Comment (jakerockland) [reply src/config.json]: Good catch, thank you!! Fixed!

**Review by gravelcycles — APPROVED** [review]

**Comment by jakerockland** [comment]
Thank you! :)

**Pull request merged**